packages/owl-bot/src/owl-bot.ts

```diff
@@ -76,15 +78,29 @@ export = (privateKey: string | undefined, app: Probot, db?: Db) => {
       if (!installation) {
         throw Error(`no installation token found for ${head.repo.full_name}`);
       }
+      const hasRunLabel = !!context.payload.pull_request.labels.filter(
+        l => l.name === OWLBOT_RUN_LABEL
+      ).length;
+
       // If the pull request is from a fork, the label "owlbot:run" must be
       // added by a maintainer to trigger the post processor.
-      // TODO(bcoe): add support for "owlbot:run" label.
-      if (head.repo.full_name !== base.repo.full_name) {
+      if (head.repo.full_name !== base.repo.full_name && !hasRunLabel) {
         logger.info(
           `head ${head.repo.full_name} does not match base ${base.repo.full_name} skipping`
         );
         return;
       }
+      // If the pull request is *not* from a fork, ignore label being added
```

Comment:
Users may want to re-run Owl Bot if:
1.  There was a transient network or cloud build error.
2.  They don't believe the results of the first run.

Reply:
Labeling now works when *not* from fork.